src/clims/services/application.py

```diff
@@ -12,14 +13,21 @@ class ApplicationService(object):
     def __init__(self):
         self.extensibles = ExtensibleService(self)
         self.substances = SubstanceService(self)
+        self.containers = ContainerService(self)
 
 
 class InversionOfControl(object):
```

Comment:
Perhaps it would make sense to name this to something less generic? I don't know exactly what that would be right now, though.

Reply:
Perhaps you're right. This service should handle all containers so the name clearly describes what it's supposed to handle at least. We can always revisit this (next six months anyways).